test/commands/init.test.ts

```diff
@@ -149,14 +150,20 @@ describe('init', () => {
     })
     .it('handles git command error')
 
-  function checkFoldersStructure(bundleName: string) {
+  function checkFoldersStructure(bundleName: string, fromhub?: boolean) {
     checkBundleFile(bundleName, CONFIG_FOLDER)
     checkBundleFile(bundleName, CONFIG_FOLDER, CONFIG_FILE)
     checkBundleFile(bundleName, BUNDLE_DESCRIPTOR_FILE_NAME)
     checkBundleFile(bundleName, 'microservices')
     checkBundleFile(bundleName, 'microfrontends')
     checkBundleFile(bundleName, 'Dockerfile')
     checkBundleFile(bundleName, '.gitignore')
+    if (!fromhub) {
```

Comment:
files should be there in any case, even if you get it from hub

Reply:
done